Tests/LibWeb/Text/expected/wpt-import/streams/readable-byte-streams/read-min.any.txt

```diff
@@ -0,0 +1,35 @@
+Summary
+
+Harness status: OK
+
+Rerun
+
+Found 24 tests
+
+23 Pass
+1 Fail
+Details
+Result	Test Name	MessagePass	ReadableStream with byte source: read({ min }) rejects if min is 0	
+Fail	ReadableStream with byte source: read({ min }) rejects if min is negative	
```

Comment:
I think this might just be a boog in the test - the IDL specifies [EnforceRange] https://streams.spec.whatwg.org/#dictdef-readablestreambyobreaderreadoptions

So we throw an error before rejecting:
![image](https://github.com/user-attachments/assets/e0239d2c-0b8f-4ba5-b70f-96f39a9bb687)

Reply:
https://github.com/web-platform-tests/wpt/pull/49085

Reply:
Hmm, nevermind I think I completely misunderstood the spec here, since it's a promise in the IDL there is a step to transform any exception to a rejection with that reason